tools/mason/MasonUtils.chpl

```diff
@@ -23,6 +23,7 @@
 use Spawn;
 use FileSystem;
 use TOML;
+use DateTime;
```

Comment:
Are we actually using the `DateTime` module or was that just for debugging?

Reply:
This was for debugging. I will remove it.